docs/maven-tutorial.md

```diff
@@ -0,0 +1,251 @@
+to point our builder image to our Nexus repository.
+
+```bash
+oc new-build openshift/wildfly-100-centos7:latest~https://github.com/bparees/openshift-jee-sample.git \
+	-e MAVEN_MIRROR_URL=http://nexus.<Nexus_Project>:8081
+oc start-build openshift-jee-sample --follow
+```
+
+> NOTE: Replace <Nexus_Project> with the project name of the Nexus repository.
+> If it is in the same project as the application that is using it, you can
+> leave it blank.
+
+Finally, read [Confirming Success](#confirming-success) to make sure everything
+is working correctly.
+
+#### Modifying the Builder Image
+
+If your image does not have an environment variable, you may need to modify the
+builder image. This section will go through the process of moving a Maven
+settings file into the builder image, which configures Maven to use the Nexus
+repository.
+
+Create a directory in your project called 'wildfly-builder'. Inside this
+direcotry, create a file in your project called `settings.xml` with the
+following content:
+```xml
+<settings>
+  <mirrors>
+    <mirror>
+      <!--This sends everything else to /public -->
+      <id>nexus</id>
+      <mirrorOf>*</mirrorOf>
+      <url>http://nexus.<Nexus_Project>:8081/content/groups/public</url>
+    </mirror>
+  </mirrors>
+  <profiles>
+    <profile>
+      <id>nexus</id>
+      <!--Enable snapshots for the built in central repo to direct -->
+      <!--all requests to nexus via the mirror -->
+      <repositories>
+        <repository>
+          <id>central</id>
+          <url>http://central</url>
+          <releases><enabled>true</enabled></releases>
+          <snapshots><enabled>true</enabled></snapshots>
+        </repository>
+      </repositories>
+     <pluginRepositories>
+        <pluginRepository>
+          <id>central</id>
+          <url>http://central</url>
+          <releases><enabled>true</enabled></releases>
+          <snapshots><enabled>true</enabled></snapshots>
+        </pluginRepository>
+      </pluginRepositories>
+    </profile>
+  </profiles>
+  <activeProfiles>
+    <!--make the profile active all the time -->
+    <activeProfile>nexus</activeProfile>
+  </activeProfiles>
+</settings>
+```
+
+This file is telling Maven to use only the repository at
+`nexus.<Nexus_Project>:8081`, which is our proxy. Replace `<Nexus_Project>` with
+the project that has the Nexus repo. If the Nexus repo is in the same project as
+the applications using it, you can leave it out. More information on DNS
+resolution in OpenShift can be found
+[here](https://docs.openshift.org/latest/architecture/additional_concepts/networking.html).
+
+Create a Dockerfile in the wildfly-builder directory with a `FROM` entry that
+points to the builder image you normally use. Add a `COPY` entry pointing to
+the `settings.xml` you created earlier.
+
+```
+FROM openshift/wildfly-100-centos7:latest
+
+COPY settings.xml /usr/share/maven/conf/settings.xml
+COPY settings.xml /opt/app-root/src/.m2/settings.xml
```

Comment:
It doesn't make much sense for the example, but that's the location that the reader should be copying it to for a default Maven usage.  It's explained a little lower, so I'll remove it from this.